distribution/lib/Standard/Base/0.0.0-dev/src/Enso_Cloud/Data_Link.enso

```diff
@@ -54,10 +47,53 @@ type Data_Link_Format
          before writing the config we try to parse it to ensure that it
          represents a valid data-link. In some cases (e.g. testing), we may want
          to skip that.
-    write_config (file : Writable_File) (config : JS_Object) (replace_existing : Boolean = False) (skip_validation : Boolean = False) =
-        checked = if skip_validation.not then Data_Link_Format.validate_config config
+    write_config (file : File_Like) (config : JS_Object) (replace_existing : Boolean = False) (skip_validation : Boolean = False) =
+        checked = if skip_validation.not then Data_Link.validate_config config
         checked.if_not_error <|
-            Data_Link_Format.write_raw_config file config.to_json replace_existing
+            Data_Link.write_raw_config file config.to_json replace_existing
+
+    ## PRIVATE
+       ADVANCED
+       Checks if the config represents a valid data-link.
+
+       If the library providing the data-link is not imported, this function
+       will fail with `Missing_Data_Link_Library`, even if the config would be
+       valid.
+    validate_config (config : JS_Object) -> Nothing ! Missing_Data_Link_Library | Illegal_State =
+        interpret_json_as_data_link config . if_not_error Nothing
+
+    ## ADVANCED
+       Copies the data-link configuration from one place to another.
+
+       The operation does not follow the data links, but copies the
+       configuration between the provided files.
+
+       Arguments:
+       - source: The file to copy the configuration from.
+       - target: The file to copy the configuration to.
+       - replace_existing: A flag specifying if the operation should replace an
+         existing file. By default, the operation will fail if the file already
+         exists.
+    copy (source : File_Like) (target : File_Like) (replace_existing : Boolean = False) -> Any ! File_Error =
+        raw_config = Data_Link.read_raw_config source
+        Data_Link.write_raw_config target raw_config replace_existing
+
+
+    ## ADVANCED
```

Comment:
What does ADVANCED do? I haven't seen it before

Reply:
Currently, it does not affect any logic.

I think the tag should be displayed in the documentation for this method.

The tags have been defined in https://github.com/enso-org/enso/blob/develop/docs/syntax/comments.md#tags but admittedly we haven't been using them too much and will probably need to revise it at some point anyway. But we were using `ADVANCED` in some places to mean stuff that is, well, for more advanced use cases.